eras/shelley/impl/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Version history for `cardano-ledger-shelley`
 
+## 1.11.0.0
```

Comment:
Looks like you never read the thing in bold in the PR where you place a checkmark :grin: 
> All visible changes are prepended to the latest section of a CHANGELOG.md for the affected packages. New section is never added with the code changes. 

In particular this one: **New section is never added with the code changes.** 

Version 1.10.0.0 has not been released yet

Reply:
Yes, sorry! My bad. I will revert all the bumps :sweat: